api/URL.json

```diff
@@ -224,7 +224,7 @@
             "deprecated": false
           }
         },
-        "no_MediaStream_argument": {
+        "no_MediaStream_parameter": {
```

Comment:
I don't approve of this name either but don't know what to suggest. It's about the lack of support for a specific type as the first parameter, not support for the parameter itself.

I think it'd probably make sense to invert this and treat it as a deprecated feature which has been removed, to put it on a path to eventual removal from BCD.

Reply:
> I think it'd probably make sense to invert this and treat it as a deprecated feature which has been removed, to put it on a path to eventual removal from BCD.

Yes, this sounds like the right approach to me and what I would've suggested myself. (If it's too fussy to deal with in this PR, then I'd suggest leaving it out of this PR entirely and opening an issue to track the idea.)

Reply:
I'll revert this.